app/javascript/components/admin/roles/RolePill.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Badge} from "react-bootstrap";
+
+export default function RolePill({ role }) {
+  const { name, color } = role
+
+  return (
+    <Badge pill ref={(el) => el && el.style.setProperty('background-color', color, 'important')}>
+      {name}
+    </Badge>
+  );
+}
```

Comment:
It may be good to include a prop type entry for the role,

Reply:
Good catch - done